src/utils/Table/utils.js

```diff
@@ -34,10 +34,12 @@ export const GenerateTableCellFromField = (id, fieldId, fieldOptions, labelId) =
                 id={id === 0 ? `${labelId}-label` : undefined}
                 align={fieldOptions.align || "right"}
             >
-                {fieldOptions?.linkDestination ?
-                    <Link to={linkDestination} className={classes.fieldLink}>
-                        {fieldOptions?.value}
-                    </Link> : fieldOptions.value}
+                <div style={{ marginBlock: extended && "20px" }}>
```

Comment:
If `extended` is `false` won't this style have a weird value? What happens in that case?

Reply:
Yes, you're right, it becomes `false`. I should have used a ternary operator. The result is the same nonetheless (the margin becomes zero).